packages/linux/package.mk

```diff
@@ -30,9 +30,10 @@ case "$LINUX" in
 esac
 
 PKG_KERNEL_CFG_FILE=$(kernel_config_path) || die
-
-if [ -n "$($ROOT/$SCRIPTS/uboot_helper $PROJECT $DEVICE $UBOOT_SYSTEM crust_config)" ]; then
+if [ -n "$UBOOT_SYSTEM" ]; then
```

Comment:
This will not work as intended since the `linux` package is not rebuilt for each unique `UBOOT_SYSTEM`, the `crust` patches must be applied to entire Allwinner project or for each device that can support `crust`.

Reply:
once #4831 is merged, UBOOT_FIRMWARE variable can be here inspected for "crust"

Reply:
Damn, I'm not sure if running a kernel with these patches but without scp firmware regresses anything, that's my I tried (and failed) to do this per board

Reply:
I think it shouldn't, but it should be tested anyway.

Reply:
According to Crust main author, Linux patches for crust don't break anything and some/most of them will be mainlined soon anyway. So this can be really simplified.